.github/workflows/ci.yaml

```diff
@@ -67,10 +67,10 @@ jobs:
       fail-fast: false
       matrix:
         juju:
-          - agent: 2.9.50
+          - agent: 2.9.50  # renovate: juju-agent-pin-minor
             libjuju: ^2
-          - agent: 3.1.9
-          - agent: 3.4.5
+          - agent: 3.1.9  # renovate: juju-agent-pin-minor
+          - agent: 3.4.5  # renovate: juju-agent-pin-minor
```

Comment:
The list here is questionable. IMHO, we should test 3.5 instead of 3.4 OR even 3.1.
@delgod do we have here some special obligations for Juju 3.1?

Any way, it can be updated later anytime.